Fun Commands/Mini-Games/Dyno-RPG/README.md

```diff
@@ -0,0 +1,56 @@
+# Dyno RPG
+Otherwise called **Avelon RPG**, Dyno RPG is a RPG Game made using Dyno CC and inspirted by the infection minigame and a few other custom commands. It is planned to be a community prject to be built into something everybody can enjoy. As such everyone is welcome to add their own commands to it as they desire.
+
+
+## Registration Commands
+This command is used to join the world of Avelon
+
+`register`: Grants the **Adenturer** role and enters you itno the forsaken world of Avelon
```

Comment:
Spelling error here, itno should be into

Reply:
ok lemme fix that rn =)

Reply:
done